swagger/v2/put_people_responses.yaml

```diff
@@ -0,0 +1,52 @@
+'200':
+  type: object
+  required:
+  - data
+  properties:
+    data:
+      $ref: '../v2/person.yaml#/Person'
+'400':
+  type: object
+  required:
+  - errors
+  properties:
+    errors:
+      type: array
+      items:
+        $ref: '../v1/errors.yaml#/BadRequest'
```

Comment:
not sure if a v2 endpoint should refer to v1 models or not ? I don't know what the write behaviour is but might be worth making an explicit note that the v2 schema uses v1 models for errors

Reply:
How would you make the explicit note?
This is transparent n the swagger docs, maybe we should move the shared elements to a /shared folder? not for now 😄, it would require a bit of work